content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,47 @@
+
+Haiku project is by no means a small project, it is a full operating system nonetheless. Maintaining a full-stack operating system is a huge work, and needs more than a handful of developers. Recently, one of our core developers, waddlesplash has recently announced his leave from the project due to personal reasons. waddlesplash has contributed to Haiku Project close to a decade, and has been the top contributor for 2017, 2018, and 2019. He has contributed invaluable additions to the Haiku Project, like the NVME driver, and USB3 support, with countless bug fixes and improvements on numerous Haiku components.
+
+Currently with his departure, we do not have the enough developer power to release R1.
+
+## Call for Developers
+
+If you are a C++ developer, we want you to help us in developing and maintaining Haiku. Our priority is to release R1, therefore we need urgent support in fixing R1 milestone bugs. Other than that, improvements on [these areas](https://www.haiku-os.org/community/getting-involved/developing) are necessary to provide a stable and usable user experience for Haiku users.
+
+## Why should I contribute to Haiku?
+
+- Haiku is open source, and licensed with a permissive license (MIT).
+- Haiku is inspired by BeOS, which is highly praised as an operating system; now, and back in the day.
+- Haiku offers a POSIX compatible operating system designed from scratch, with no legacy burden that other operating systems suffer from.
+- The Haiku experience does not suffer from fragmentation. All Haiku components are maintained in a single codebase, without having to rely on other teams for core system components. No flame wars between Z12 and Roadcountry supporters, or Leprechauns and Krusaders. We don't curse on the mailing lists as well.
+- Haiku is a GUI-first operating system, unlike most other FOSS systems. The Terminal is still a first-class citizen after booting the system, right there waiting for you.
+- Haiku is the only operating system that completely stays true to the [desktop metaphor](https://en.wikipedia.org/wiki/Desktop_metaphor). Tracker is the only remaining mainstream spatial desktop manager. If you are looking for an desktop experience close to the Classic Mac OS, Haiku is your guy.
+- Haiku offers a unique solution for package management, which most Linux distributions seem to suffer from. All Haiku packages are mountable disk images, therefore they do not litter your system with copied files everywhere. Updating and recovering the system is as easy as replacing a single file.
+- You might find it surprising that we respect the "everything-is-a-file" concept more than other \*nixes.
```

Comment:
I'm surprised by this statement. Source?

Reply:
Which one?

Reply:
this one: "we respect the "everything-is-a-file" concept more than other \*nixes"

Reply:
It's a nod to hosting e-mails, bookmarks, contacts etc. as files.

Reply:
The concept is accurately: Everything is a file descriptor. Referring to it can be a misunderstanding imo

Reply:
Removed it.